content/server-server-api.md

```diff
@@ -1245,6 +1244,22 @@ The following endpoint prefixes MUST be protected:
 -   `/_matrix/federation/v1/event_auth`
 -   `/_matrix/federation/v1/get_missing_events`
 
+Additionally the [`/_matrix/federation/v1/send/{txnId}`](#put_matrixfederationv1sendtxnid)
+endpoint MUST be protected as follows:
+
+-   ACLs MUST be applied to all PDUs on a per-PDU basis. If the `origin`
```

Comment:
We should be making the `origin` field go away rather than doubling-down on its existence (https://github.com/matrix-org/matrix-spec/issues/374).

Can you soften this language to "sending server" or somesuch?

Reply:
If anything, it would be "the `origin` field of the `X-Matrix` `Authorization` header", but it gets quite cumbersome, and IMHO somewhat obvious.

We could maybe link to https://spec.matrix.org/unstable/server-server-api/#request-authentication, but the place to do so would be once near the top of the ACLs section, rather than down here.